_shows/20_21/freshers_fringe.md

```diff
@@ -0,0 +1,167 @@
+- role: (Tara, Oliver and Nicholas' Group)
+  name: Ylana Gibert
+- role: (Helena, Sofia and Katie's Group)
+  name: Demi Idowu
+- role: (Helena, Sofia and Katie's Group)
+  name: Lelah Gorgin
+- role: (Helena, Sofia and Katie's Group)
+  name: Luci Dodd
+- role: (Abie and Georgia's Group)
+  name: Cigdem Celiker
+- role: (Abie and Georgia's Group)
+  name: Abraham Botha
+- role: (Alexandra and Rachel's Group)
+  name: Megan Jackman
+- role: (Alexandra and Rachel's Group)
+  name: Rachel Allcock
+- role: (Alexandra and Rachel's Group)
+  name: Talia Krais
+- role: (Jake and Libby's Group)
+  name: Iain Messore
+- role: (Jake and Libby's Group)
+  name: Maria Mcdonagh
+- role: (Jake and Libby's Group)
+  name: Georgie Bullen
+- role: (Joe and Lousie's Group)
+  name: Isabel Johnson
+- role: (Joe and Lousie's Group)
+  name: Anna Bulcock
+- role: (Joe and Lousie's Group)
+  name: Alice Clapham
+- role: (Jacob and Charlotte's Group)
+  name: Freya Howcutt
+- role: (Jacob and Charlotte's Group)
+  name: Daisy Miles
+- role: (Cheidza and Lucy's Group)
+  name: Sophie Lagden
+- role: (Cheidza and Lucy's Group)
+  name: Amelia Fleming
+- role: (Laura and Robert's Group)
+  name: Naomi Hallowell
+- role: (Laura and Robert's Group)
+  name: Jeremy Grau
+- role: (Barney, Sofia and Hettie's Group)
+  name: Charlotte Armitage
+- role: (Barney, Sofia and Hettie's Group)
+  name: Tom Taylor 
+- role: (Barney, Sofia and Hettie's Group)
+  name: Hatty McGrath  
+- role: (Cecilia and Jesse's Group)
+  name: Priya Joshi
+- role: (Cecilia and Jesse's Group)
+  name: James Appleby
+- role: (Cecilia and Jesse's Group)
+  name: Victoria Inzari
+  
+crew: 
+- role: Executive Producer
+  name: Caetano Capurro
+- role: Director 
+  name: Caetano Capurro
+- role: Director 
+  name: Ella Seber Rajan
+- role: Director 
+  name: Charlie Basley
+- role: Director 
+  name: Nadia Elalfi
+- role: Director 
+  name: Alice Walker
+- role: Director 
+  name: Caitie Pardoe
+- role: Director 
+  name: Gabriella Teriaca
+- role: Director 
+  name: Tara Anegada
+- role: Director 
+  name: Oliver Binns
+- role: Director 
+  name: Nicholas Landon
+- role: Director 
+  name: Helena Hunt
+- role: Director 
+  name: Sofia Loreti
+- role: Director 
+  name: Katie McCabe
+- role: Director 
+  name: Alexandra Hogg
+- role: Director 
+  name: Rachel Coussins
+- role: Director 
+  name: Jake Levy
+- role: Director 
+  name: Libby Horobin
+- role: Director 
+  name: Joe Butler
+- role: Director 
+  name: Louise Tallon
+- role: Director 
+  name: Jacob Gausden
+- role: Director 
+  name: Charlotte Smith
+- role: Director 
+  name: Cheidza McNab
+- role: Director 
+  name: Lucy House
+- role: Director 
+  name: Laura Sherratt
+- role: Director 
+  name: Robert Goodall
+- role: Director 
+  name: Barney Hartwill
+- role: Director 
+  name: Sofia Bassani
+- role: Director 
+  name: Hettie Rockell
+- role: Director 
+  name: Cecilia Alexander
+- role: Director 
+  name: Jesse Rough
```

Comment:
No one technical listed? Did directors edit their own? Etc

Reply:
i could not find credits for this role

Reply:
Added in crew_incomplete